types/uuid/interfaces.d.ts

```diff
@@ -48,3 +48,17 @@ export interface v5Static {
     URL: string;
 }
 export type v5 = v5Buffer & v5String & v5Static;
+
+export type NIL = string;
+
+export enum Version {
```

Comment:
This object doesn't exist at runtime; it should not be an enum. It'd actually be nice to remove this file entirely and fold it into the parent file, since there's no `uuid/interfaces` module

Reply:
OK, you are right about both, the enum and inlining the interface definitions (I inherited that structure and – being rather new to TypeScript typings – never questioned it).

If I inline `uuid/interfaces` how do I go about only `export`ing the named exports that are actually exported by the `uuid` module? I'm getting

```
All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting.
```

errors if I don't export all types. I've seen https://stackoverflow.com/a/51970377/1053532 but what is preferred in DefinitelyTyped? Adding `export {}` or declaring a `namespace`?

Pointing me to a module where this is done correctly would be enough, I can then make the change.

Reply:
@RyanCavanaugh friendly ping

Reply:
FWIW I went for the `export {}` route.